en/core-api/e-wallet.md

```diff
@@ -302,49 +351,73 @@ You will get the **API response** like the following.
+| status_message     | A message describing the status of the transaction.          | String | --                                                           |
+| transaction_id     | The *Transaction ID* of the specific transaction.            | String | --                                                           |
+| order_id           | The specific *Order ID*.                                     | String | --                                                           |
+| gross_amount       | The total amount of transaction for the specific order.      | String | --                                                           |
+| currency           | The unit of currency used for the transaction.               | String | --                                                           |
+| payment_type       | The type of payment method used by the customer for the transaction. | String | --                                                           |
+| transaction_time   | The date and time at which the transaction occurred.         | String | It is in the format, *YYYY-MM-DD* *HH:MM:SS.*<br>Time zone: Western Indonesian Time (GMT+7) |
+| transaction_status | The status of the transaction.                               | String | For more details, refer to [Transaction Status](/en/after-payment/get-status.md#transaction-status). |
+| fraud_status       | The fraud_status of the transaction is displayed.            | String | --                                                           |
+| actions            | The stage of the transaction                                 | Array  | There are four stages of the transaction that can be retrieved through this attribute: QR code, Deeplink, Status of transaction, Canceling the transaction |
+| name               | The name of the action.                                      | String | --                                                           |
+| method             | The type of the method.                                      | String | --                                                           |
+| url                | The redirect url                                             | String | --                                                           |
 
-The user flow varies when using a web browser (on a computer or a tablet) compared to a SmartPhone.
+</article>
+</details>
 
-### Show QR Code Image (on computer or tablet)
+### 2. Altering payment flow depending on the device used
 
-To display transaction QR Code image, use the url from `generate-qr-code` actions retrieved from API response. Simplest way is to **"hotlink"** the image url, if the frontend is HTML, put the url in image tag `<img src="[QR CODE URL]">`, or display it on a similar component without downloading.
-If the frontend does not support such scenario, download the QR code image from that url, then display it on frontend.
+The user flow is different for a web browser (on a computer or a tablet) and a smartphone.
```

Comment:
> The user flow is different for a web browser (on a computer or a tablet) and a smartphone.

This sentence seems off, web browser can also be opened from smartphone. This sentence should explain that "the flow of payment" should be treated differently between when a customer is transacting Smartphone and non-smartphone. Can you improve the sentence?